pl_bolts/models/detection/yolo/utils.py

```diff
@@ -0,0 +1,155 @@
+from typing import List
+
+import torch
+from packaging import version
+from torch import Tensor
+
+from pl_bolts.utils import _TORCHVISION_AVAILABLE
+from pl_bolts.utils.warnings import warn_missing_pkg
+
+if _TORCHVISION_AVAILABLE:
+    from torchvision.ops import box_iou
+else:
+    warn_missing_pkg("torchvision")
+
+# PyTorch 1.10 introduced the argument "indexing" and deprecated calling without the argument. Since we call it inside
+# a "@torch.jit.script" function, it's difficult to make this decision at call time.
+if version.parse(torch.__version__) >= version.parse("1.10.0"):
```

Comment:
This looks like it's throwing an error in one of the tests, please use `_compare_version` from `pl_bolts.utils`

Reply:
This now uses _TORCH_MESHGRID_REQUIRES_INDEXING, as it was fixed in the other pull request.